buildSrc/src/main/kotlin/OpenApiPlugin.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.apply
+import org.projectnessie.buildtools.smallryeopenapi.SmallryeOpenApiPlugin
+
+/** Makes the generated sources available to IDEs, disables Checkstyle on generated code. */
+@Suppress("unused")
+class OpenApiPlugin : Plugin<Project> {
```

Comment:
is this split-out intentionally included in this PR ?
there doesnt seem any mention of it in the commit msg

Reply:
yea - it's completely going "away" w/ #5936